server/apps/distributed-app/docs/modules/ROOT/partials/RemoteDelivery.adoc

```diff
@@ -66,6 +66,7 @@ within the mailet code.<br>
 Note: This facility should be used with extreme care by expert users with a thorough knowledge of the relevant RFCs and
 the ability to perform their own problem resolutions.
 * *debug* (optional) - a Boolean (true/false) indicating whether debugging is on. Default is false.
+* *onSuccess* (optional) - if specified, this processor is called for each email successfully sent to remote third parties.k
```

Comment:
Typo at the end: parties.**k**

Reply:
Maybe move this up in the list so it is next to "bounceProcessor", for context.

Reply:
Maybe add a hint to the end of the documentation:
"When using bounceProcessor or onSuccess processors, take special care of error handling (see onMailetException and onMatcherException) to avoid confusing situations. Also remember that on partial delivery, both processors will be used: onSuccess with successfull recipients, and bounceProcessor with failed recipients."